package-lock.json

```diff
@@ -0,0 +1,13271 @@
+{
```

Comment:
@aulneau my assumption is we should not commit this change as part of a content update. correct?

Reply:
This file should not be here, you are using npm where you should use `yarn`

Reply:
Oops, got it. `npm` is a habit for me. Should we remove support from the repo if `yarn` is preferred? 

I'll remove this file meanwhile.